package.json

```diff
@@ -22,7 +22,7 @@
     "build-storybook": "build-storybook --config-dir .storybook --output-dir .out --static-dir ./public",
     "clean": "rimraf dist lib src/icons/**/svgs",
     "build": "run-s clean && run-s build:icons:all && run-s tsc && run-p build:rollup build:babel",
-    "build:icons:all": "run-s build:icons build:icons:index",
+    "build:icons:all": "run-s clean && run-s build:icons build:icons:index",
```

Comment:
I think u put this here while you were testing your changes, but this is redundant in the build script since it already run clear as the first script task in the general build

Reply:
Needing to run all of `build` when just rebuilding icons (when making changes to just icons) takes 30+ more seconds than is necessary. It's okay to run clean twice on `build`.